src/util.ts

```diff
@@ -13,6 +13,17 @@ const suffixes: Record<string, string> = {
 	other: 'th'
 };
 
+export const banMessageDeleteChoices: Array<APIApplicationCommandOptionChoice<number>> = [
+	{ name: 'Previous 30 Minutes', value: 30 * 60 },
+	{ name: 'Previous Hour', value: 60 * 60 },
+	{ name: 'Previous 3 Hours', value: 3 * 60 * 60 },
+	{ name: 'Previous 6 Hours', value: 6 * 60 * 60 },
+	{ name: 'Previous 12 Hours', value: 12 * 60 * 60 },
+	{ name: 'Previous Day', value: 24 * 60 * 60 },
+	{ name: 'Previous 3 Days', value: 3 * 24 * 60 * 60 },
+	{ name: 'Previous Week', value: 7 * 24 * 60 * 60 },
+];
```

Comment:
Shouldn't this be part of https://github.com/PretendoNetwork/chubby/pull/39? I feel like something has gone wrong with this git history